src/node/stage/approved.rs

```diff
@@ -2105,7 +2079,7 @@ impl Approved {
         Ok(())
     }
 
-    fn send_send_neighbour_info(&mut self) -> Result<()> {
+    fn send_neighbour_info(&mut self, core: &mut Core) -> Result<()> {
```

Comment:
As discussed elsewhere, I don't think we need this at all.

Reply:
will have a double check once we settled with the other discussion.

Reply:
covered by routing PR https://github.com/maidsafe/routing/pull/2170